### PR TITLE
Fe updates

### PR DESCRIPTION
**Pull request by andysellick**
## What

- update the analytics documentation with details of the GA4 data for events on the back link component
- update the local frontend development page with a workaround to a newly discovered problem

## Why
Needed for the GA4 work.

Trello card: https://trello.com/c/Z5FYBU5o/782-back-links

### CODE REVIEW

File: source/manual/local-frontend-development.html.md [hunk 1]
```diff
@@ -128,6 +128,22 @@ Some components, such as the cookie banner, are pulled in by Static. To test cha
 
 ## Troubleshooting
 
+### JavaScript changes not appearing/JavaScript not working properly
+
+Sometimes the main JavaScript files cannot be served properly when running locally. This happens when Rails automatically inserts fingerprinting into the script URLs when it is not needed e.g. `application-834y2r2hr292y9r.js` is being requested, but only `application.js` exists.
+
+This problem does not currently have a solution, but a workaround is to modify `_layout_for_public.html.erb` in your local `govuk_publishing_components` to hardcode the URLs to the broken JavaScript files (`application.js`, `load-analytics` and any others you need). For example, change:
```
Comment (AshGDS) [inline source/manual/local-frontend-development.html.md:135]: Sorry to be nitpicky here, but regarding `This problem does not currently have a solution` -  have we asked the GOVUK devs if they've also experienced this but had a solution for it? Just a bit mindful of documenting  that there's no solution if we haven't had this confirmed.

File: data/analytics/events.yml [hunk 2]
```diff
@@ -76,6 +76,28 @@
         - name: external
         - name: method
         - name: link_domain
+
+- name: back link
+  description:
+  events:
+    - name: link click
+      description: When the back link is used, on pages like the email subscription journeys
```
Comment (AshGDS) [inline data/analytics/events.yml:84]: Is it worth using the `examples` value here to provide a link to an example?

**Review by AshGDS — COMMENTED** [review]
Thanks for doing this - had a couple small comments

File: source/manual/local-frontend-development.html.md (hunk 1 above)
Comment (andysellick) [reply source/manual/local-frontend-development.html.md]: Reasonable point. I've reworded for now.

File: data/analytics/events.yml (hunk 2 above)
Comment (andysellick) [reply data/analytics/events.yml:84]: Yeah I wanted to but linking directly into the subscriptions would only work if someone was signed in. There's got to be other examples around the site but I can't find any right now. I'm hoping once the tracking is enabled we'll be able to put an actual example in here.

Comment (AshGDS) [reply data/analytics/events.yml:84]: No worries 👍

File: source/manual/local-frontend-development.html.md (hunk 1 above)
Comment (AshGDS) [reply source/manual/local-frontend-development.html.md]: Thanks 👍

**Review by AshGDS — APPROVED** [review]
🚀

**Pull request merged**